docs/source/usersguide/materials.rst

```diff
@@ -0,0 +1,169 @@
+.. _usersguide_materials:
+
+.. currentmodule:: openmc
+
+=====================
+Material Compositions
+=====================
+
+Materials in OpenMC are defined as a set of nuclides/elements at specified
+densities and are created using the :class:`openmc.Material` class. Once a
+material has been instantiated, nuclides can be added with
+:meth:`Material.add_nuclide` and elements can be added with
+:meth:`Material.add_element`. Densities can be specified using atom fractions or
+weight fractions. For example, to create a material and add Gd152 at 0.5 atom
+percent, you'd run::
+
+   mat = openmc.Material()
+   mat.add_nuclide('Gd152', 0.5, 'ao')
+
+The third argument to :meth:`Material.add_nuclide` can also be 'wo' for weight
+percent. The densities specified for each nuclide/element are relative and are
+renormalized based on the total density of the material. The total density is
+set using the :meth:`Material.set_density` method. The density can be specified
+in gram per cubic centimeter, atom per barn-cm, or kilogram per cubic meter,
```

Comment:
Consider enclosing the string formmated units in parentheses following each unit that (*e.g.*, `'g/cm3'`, `'at/b-cm'` and `'kg/m3'`).